tests/_test_msui/test_multiple_flightpath_dockwidget.py

```diff
@@ -25,33 +25,78 @@
     limitations under the License.
 """
 import pytest
-from PyQt5 import QtTest
+from PyQt5 import QtTest, QtWidgets, QtCore, QtGui
+from mock import mock
+
 from mslib.msui import msui
-from mslib.msui.multiple_flightpath_dockwidget import MultipleFlightpathControlWidget
-from mslib.msui import flighttrack as ft
+# from mslib.msui.multiple_flightpath_dockwidget import MultipleFlightpathControlWidget
+# from mslib.msui import flighttrack as ft
 import mslib.msui.topview as tv
 
 
 class Test_MultipleFlightpathControlWidget:
     @pytest.fixture(autouse=True)
     def setup(self, qtbot):
-        self.window = msui.MSUIMainWindow()
-        self.window.create_new_flight_track()
-
-        self.window.actionNewFlightTrack.trigger()
-        self.window.actionTopView.trigger()
-        initial_waypoints = [ft.Waypoint(40., 25., 0), ft.Waypoint(60., -10., 0), ft.Waypoint(40., 10, 0)]
-        self.waypoints_model = ft.WaypointsTableModel("myname")
-        self.waypoints_model.insertRows(
-            0, rows=len(initial_waypoints), waypoints=initial_waypoints)
-
-        self.widget = tv.MSUITopViewWindow(model=self.waypoints_model, mainwindow=self.window)
-        self.window.show()
-        QtTest.QTest.qWaitForWindowExposed(self.window)
+        # Start a MSUI window
+        self.main_window = msui.MSUIMainWindow()
+        self.main_window.show()
+        qtbot.wait_exposed(self.main_window)
+
+        # Create two flight tracks
+        self.main_window.actionNewFlightTrack.trigger()
+        self.main_window.actionNewFlightTrack.trigger()
+
+        # Open a Top View window
+        self.main_window.actionTopView.trigger()
+        self.topview_window = self.main_window.listViews.currentItem().window
+
+        # Switch to the Multiple Flightpath Widget
+        self.topview_window.cbTools.setCurrentIndex(6)
+
+        # Get a reference to the created MultipleFlightpathControlWidget
+        self.multiple_flightpath_widget = self.topview_window.docks[tv.MULTIPLEFLIGHTPATH].widget()
```

Comment:
There is no need to re-use this class. The class is a remnant from before MSS used pytest and I would prefer not to use this style of test definition. Test can just be simple functions. If you want to extract the setup part into something separate to use in multiple test functions, then just define a pytest fixture for that.